Platform/ElkhartlakeBoardPkg/AcpiTables/SsdtRtd3/EhlCrbRtd3.asl

```diff
@@ -116,44 +119,9 @@ External(WWAN_PCIE_ROOT_PORT.LASX)
       Store(PSWP, WAKG)
       Name(WAKP, 0)             // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
       Name(SCLK, 0)
+      Store(PS1C, SCLK)
       Include("Rtd3Pcie.asl")
-    }
-  //}
-
-    ///
-    /// PCIe RP09 RTD3 - M.2 SSD RTD3
-    ///
-    Scope(\_SB.PC00.RP09) {
-      Name(RSTG, Package() {0, 0})
-      Store(SSDR, Index(RSTG, 0))
-      Store(SDRP, Index(RSTG, 1))
-      Name(PWRG, Package() {0, 0})
-      Store(SSDP, Index(PWRG, 0))
-      Store(SDPP, Index(PWRG, 1))
-      Name(WAKG, 0)              // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
-      Name(WAKP, 0)              // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
-      Name(SCLK, 3)
-      Include("Rtd3Pcie.asl")
-      Scope(\_SB.PC00.RP09.PXSX) {
-        Include("Rtd3PcieSsdStorage.asl")
-      }
-    }
-
-    ///
-    /// PCIe RP13 RTD3 - M.2 SSD RTD3
-    ///
-    Scope(\_SB.PC00.RP13) {
-      Name(RSTG, Package() {0, 0})
-      Store(SD2R, Index(RSTG, 0))
-      Store(SDR1, Index(RSTG, 1))
-      Name(PWRG, Package() {0, 0})
-      Store(SD2P, Index(PWRG, 0))
-      Store(SDP1, Index(PWRG, 1))
-      Name(WAKG, 0)             // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
-      Name(WAKP, 0)             // must be defined due to compiler bug, will be removed when fixed https://bugs.acpica.org/show_bug.cgi?id=1432
-      Name(SCLK, 0)
-      Include("Rtd3Pcie.asl")
-      Scope(\_SB.PC00.RP13.PXSX) {
+      Scope(\_SB.PC00.RP01.PXSX) {

```

Comment:
RP09 and RP13 do not require RTD asl?

Reply:
Yes, it is not require and BIOS doesn't have these too.